src/m365/spo/commands/site/site-commsite-enable.ts

```diff
@@ -54,20 +64,36 @@ class SpoSiteCommSiteEnableCommand extends SpoCommand {
     this.validators.push(
       async (args: CommandArgs) => {
         if (args.options.designPackageId &&
-          !validation.isValidGuid(args.options.designPackageId)) {
-          return `${args.options.designPackageId} is not a valid GUID`;
+          !this.isValidDesignPackageId(args.options.designPackageId)) {
```

Comment:
I think it is better if we only validate if it's a valid guid here. Who knows: MS might add new id's, which would lead to our command immediately being behind the times.

The `designPackage` option may be validated strictly, that's useful and just. The `designPackageId` option may be less strict.